compass/ocean/tests/global_ocean/mesh/wc14/dynamic_adjustment/__init__.py

```diff
@@ -221,6 +221,7 @@ def __init__(self, test_group, mesh, init, time_integrator):
 
         step.add_input_file(filename='../{}'.format(restart_filenames[5]))
         step.add_output_file(filename='../{}'.format(restart_filenames[6]))
+        step.add_output_file(filename='output.nc')
```

Comment:
@xylar, I'm sorry if I'm being dense here, but it seems like you are adding the output.nc file to the expected  output?  I thought this was my problem in the WC14 spin up that it didn't actually create this file.  It seems it will now be expected?  I feel like i'm missing something obvious so sorry for the stupid question.

Reply:
I can definitely understand the confusion.  In the `simulation` step, `output.nc` is always produced and we need this for validation.  In other dynamic-adjustment steps, it is not.

Reply:
Oh, I see, I didn't notice this was just the `simulation` step.  I now see that the file above added `output.nc` to every step for global forward now it is explicit only for simulation.  Thanks for the help understanding this!